spec/unit/controllers/runtime/spaces_controller_spec.rb

```diff
@@ -1269,120 +1270,23 @@ def decoded_guids
       let(:isolation_segment_model) { IsolationSegmentModel.make }
       let(:organization) { Organization.make }
       let(:space) { Space.make(organization: organization) }
+      let(:assigner) { IsolationSegmentAssign.new }
 
       context 'associating an isolation_segment' do
-        context 'when the space contains no apps' do
```

Comment:
@SocalNick I'm not sure who's in the queue to pick this up, but hopefully we've answered the question here?